front/cypress/e2e/admin/homepage_builder.cy.ts

```diff
@@ -76,7 +76,7 @@ describe('Homepage builder', () => {
     });
   });
 
-  it('updates homepage builder content correctly', () => {
+  it('updates and delete homepage builder content correctly', () => {
```

Comment:
Combined update and remove in one file to make sure all tests are independent from one another.